### PR TITLE
Add pix2pix example 

### PR DESCRIPTION
**Pull request by taizan**
added pix2pix implementation to examples

### CODE REVIEW

**Comment by rcalland** [comment]
Please consider merging my pull request first https://github.com/pfnet-research/chainer-pix2pix/pull/6
It contains an english translation of the readme, and a bugfix for image resizing

**Comment by taizan** [comment]
jenkins test this please

**Comment by taizan** [comment]
jenkins test this please

**Comment by kmaehashi** [comment]
Jenkins, test this please.
(@taizan Sorry, only members in Chainer organization can trigger that.)

**Comment by niboshi** [comment]
I restarted travis.

File: examples/pix2pix/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ - `unzip CMP_facade_DB_base.zip`
+3. `python train_facade.py -g [GPU ID, e.g. 0] -i [dataset root directory] --out [output directory] --snapshot_interval 10000`
+4. Wait a few hours...
+ - `--out` stores snapshots of the model and example images at an interval defined by `--snapshot_interval`
+ - If the model size is large, you can reduce `--snapshot_interval` to save resources.
+
+# Using other datasets
+- Gather image pairs (e.g. label + photo). Several hundred pairs are required for good results.
+- Create a copy of `facade_dataset.py` for your dataset. The function get_example should be written so that it returns the i-th image pair a tuple of numpy arrays i.e. `(input, output)`.
+- It maybe necessary to update the loss function in `updater.py`.
```
Comment (bkvogel) [inline examples/pix2pix/README.md]: It maybe -> It may be

**Comment by bkvogel** [comment]
I think it looks good other than a small typo.

**Comment by niboshi** [comment]
@taizan
After following review, please merge master branch.
(usually this is done automatically in CI. but Travis uses travis.yml as-is from PR branch.)

**Comment by niboshi** [comment]
jenkins, test this please

**Comment by bkvogel** [comment]
Thank you for the changes.
jenkins, test this please

**Comment by bkvogel** [comment]
LGTM!

**Pull request merged**